runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -83,10 +83,33 @@ default Color getMagePrayerColor()
 		description = "Configures how many ticks should pass between fake XP drops, 0 to disable",
 		position = 4
 	)
-	@Units(Units.TICKS)
```

Comment:
this is wrong.

Reply:
This was in the original file, was it wrong to move it or should it just not have been there in the first instance?

Reply:
It was correct originally, you are requesting its removal, which is not correct.